app/routes/app/brainstorm/s/$id.tsx

```diff
@@ -0,0 +1,147 @@
+        <div className="flex flex-wrap gap-x-8 gap-y-4">
+          <Detail>
+            <Detail.Title>Sponsor</Detail.Title>
+            <Author.Author />
+          </Detail>
+          <Detail>
+            <Detail.Title>Created At</Detail.Title>
+            <Badge color="gray" size="lg">
+              <Text size="sm" className="normal-case font-normal">
+                1 month 5 days ago
+              </Text>
+            </Badge>
+          </Detail>
+          <Detail>
+            <Detail.Title>Overall Score</Detail.Title>
+            <div className="flex rounded-full bg-[#6993FF] items-center w-28">
+              <div className="rounded-full bg-[#D1DEFF] w-3/4">
+                <Text align="center" className="normal-case font-normal">
+                  Good
+                </Text>
+              </div>
+              <Text color="white" className="mx-auto pl-1 pr-2">
+                80
+              </Text>
+            </div>
+          </Detail>
+          <Detail>
+            <Detail.Title>Reviews</Detail.Title>
+            <Badge color="gray" size="lg">
+              99
+            </Badge>
+          </Detail>
+          {isWinner ? (
+            <Detail>
+              <Detail.Title>Winner</Detail.Title>
+              <Badge color="yellow" size="lg" leftSection={<Avatar size={20} src="/img/trophy.svg" />}>
+                <Text color="dark" size="sm" className="normal-case font-normal">
+                  100 SOL
+                </Text>
+              </Badge>
+            </Detail>
+          ) : (
+            <></>
+          )}
+        </div>
+        <Text color="dimmed" className="max-w-2xl">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ac augue interdum mattis elit quam sapien tellus
+          pellentesque. Vel magna consectetur mauris eu. Mauris arcu diam dolor ut tincidunt. Sit euismod sit fermentum,
+          consequat maecenas. Ante odio eget nunc velit id volutpat. Aliquam leo non viverra metus, ligula commodo
+          aliquet velit massa. Lacinia lacus amet massa
+        </Text>
+      </section>
+      <section>
+        <Title order={3}>Reviews ({reviews.length})</Title>
+        <Divider />
+      </section>
+
+      <section className="mt-3">
+        <div className="flex flex-col-reverse md:flex-row space-y-reverse space-y-7 md:space-y-0 space-x-0 md:space-x-5">
+          <main className="flex-1">
+            <div className="space-y-5">
+              <div className="overflow-auto">
+                <div className="min-w-[350px] w-full border-spacing-4 border-separate">
+                  <div className="space-y-4">
+                    {reviews.map((m) => {
+                      return (
+                        <Link
+                          to="/u/[uId]"
+                          className="flex border-solid border-2 border-[#EDEDED] py-3 px-4 rounded-lg hover:bg-stone-100 items-center space-between"
```

Comment:
Thinking that if you column-ize this whole card for mobile it will look better. `flex-col md:flex-row`. Plus some spacing of course.

Reply:
<img width="213" alt="image" src="https://user-images.githubusercontent.com/84731260/200358696-928eba54-e78c-4bba-93ba-f869365c52c1.png">